### DISCUSSION TITLE
Can I push my code to ComfyUI codebase repository?

### DISCUSSION THREAD

**Discussion by hopeman1986** [comfyanonymous/ComfyUI#4001] [General]
Hi

I want to contribute to the awesome ComfyUI Git codebase

so I just made some changes to the comfyui codebase for my custom workflow structure and I want to push the current codebase to ComfyUI Git

Can I push my codebase to ComfyUI?
How can i become a contributor, too ?

Please help me
Thanks

**Comment by BrechtCorbeel** [comment]
Yes you can, but as with anything if it is not approved it will not come through.

**Comment by hopeman1986** [comment]
Thank you so much for your answer

Sorry, but would you please tell me how to get it approved?

Thanks

**Reply by BrechtCorbeel** [reply]
By pushing it and by talking to other developers whether or not it is a worthwhile push, probably a good idea to take some input.

**Reply by BrechtCorbeel** [reply]
maybe making your own repo and then asking to integrate and deprecating the package when needed.

**Comment by hopeman1986** [comment]
ok got it.
thank you so much for your answer
